lib/at_cmd_parser/at_utils.h

```diff
@@ -255,21 +255,31 @@ static inline bool is_command(const char *str)
  */
 static bool is_clac(const char *str)
 {
+	int base = 0;
+
+	/* skip leading <CR><LF>, if any, as check not from index 0 */
+	if (is_lfcr(str[base])) {
+		base++;
```

Comment:
Actually, you could just move the str pointer (`str++`). This way you could avoid the `base` variable and other changes.

Reply:
Taken with gratitude